notebooks/tutorials/landslides/archive/Synthetic_depth_LandlabLandslide_v2-2020421.ipynb

```diff
@@ -0,0 +1,2656 @@
+{
```

Comment:
I suspect that all files in this archive folder should be deleted. @ChristinaB can you provide some context as to why they are being brought into the main repository?

Reply:
Thanks!  I can make those fixes.